pytest/tests/stress/stress.py

```diff
@@ -296,107 +293,67 @@ def revert_txs():
 
-                            for k in range(j + 1, len(last_tx_set)):
-                                tx = last_tx_set[k]
-                                expected_balances[tx[1]] += tx[4]
-                                expected_balances[tx[2]] -= tx[4]
+            with nonce_lock:
+                tx = sign_payment_tx(nodes[from_].signer_key, 'test%s' % to,
+                                     amt, nonce_val.value,
+                                     base58.b58decode(hash_.encode('utf8')))
 
-                                if get_balances() == expected_balances:
-                                    trace_reverted_txs(last_tx_set, [i, j, k])
+                # Loop trying to send the tx to all the validators, until at least one receives it
+                tx_hash = None
+                for send_attempt in range(SEND_TX_ATTEMPTS):
+                    shuffled_validator_ids = [x for x in validator_ids]
+                    random.shuffle(shuffled_validator_ids)
+                    for validator_id in shuffled_validator_ids:
+                        try:
+                            info = nodes[validator_id].send_tx(tx)
+                            if 'error' in info and info['error']['data'] == 'IsSyncing':
+                                pass
 
-                                expected_balances[tx[1]] -= tx[4]
-                                expected_balances[tx[2]] += tx[4]
+                            elif 'result' in info:
+                                tx_hash = info['result']
+                                break
 
-                            tx = last_tx_set[j]
-                            expected_balances[tx[1]] -= tx[4]
-                            expected_balances[tx[2]] += tx[4]
+                            else:
+                                assert False, info
 
-                        tx = last_tx_set[i]
-                        expected_balances[tx[1]] -= tx[4]
-                        expected_balances[tx[2]] += tx[4]
+                        except (requests.exceptions.ReadTimeout,
+                                requests.exceptions.ConnectionError):
+                            if not network_issues_expected and not nodes[
+                                    validator_id].mess_with:
+                                raise
 
-                    logging.info(
-                        "The latest and greatest stats on successful/failed: %s/%s"
-                        % (good, bad))
-                    assert False, "Balances didn't update in time. Expected: %s, received: %s" % (
-                        expected_balances, get_balances())
-            last_iter_switch = time.time()
+                    if tx_hash is not None:
+                        break
 
-        if mode == 0:
-            # do not send more than 50 txs, so that at the end of the test we have time to query all of them. When #2195 is fixed, this condition can probably be safely removed
-            if tx_count < 50:
```

Comment:
Again, github fails to display the change here correctly. This line is removed, the code around is unchanged.

Reply:
@SkidanovAlex FYI, when you review the changes, you can configure Github to hide whitespace changes and then Github displays the check correctly.

![image](https://user-images.githubusercontent.com/304265/90442856-ad53b980-e0b1-11ea-9ac7-f1dbcd3f353e.png)

Reply:
@frol that's great to know. Though given I don't know how others have github configured it's still worth flagging such places :)
But I will make sure to include these instructions next time this happens.